docs/source/reference/yaml-spec.rst

```diff
@@ -80,6 +80,12 @@ Available fields:
       #   high: 6000 IOPS; 340 MB/s; write 250 MB/s
       disk_tier: 'medium'
 
+      # Ports to expose (optional).
+      # Currently only TCP protocol is supported.
+      ports:
+        - 8080
+        - 10022
```

Comment:
For convenience, I would prefer to also support the following way as well:
```
ports:
  - 8080-8088
```

Wdyt?

Reply:
Agreed. Another thing I'm thinking about is shall we support this in CLI? Since to represent a list in CLI is a little bit tricky and IIUC is conflicted with `entrypoint` argument

Reply:
Added now